### DISCUSSION TITLE
Use mermaid-cli instead of website for diagram creation

### DISCUSSION THREAD

**Discussion by cdwillie76** [Zuehlke/poinz#286] [Ideas]
As a user building poinz on a standalone network
I would like the build to complete without errors
So that I have don't have to comment out portions of the build process for it to finish

So I have been working with the poinz code base to build a container that can run on our internal container hosting platform.  In order to do that, I need to be able to build it without connecting to the internet.  We proxy getting the node packages and I have been able to create a custom Docker file to make our security people happy.  The one part of the build I had to remove was the docu command since it tries to hit the internet directly.  Doing some research, I see a mermaid-cli package that could possibly be used instead.

Is this something you would be interested in adding to the code base?  If so, I can try to get it integrated and submit a pull request (or wait if you think it is something you could integrate quickly).

Thanks for developing and maintaining this great tool!

**Comment by xeronimus** [comment]
done in https://github.com/Zuehlke/poinz/issues/288

**Reply by cdwillie76** [reply]
Thank you!
